src/server.js

```diff
@@ -83,41 +94,90 @@ const createServer = async (port, isApi) => {
   await server.register([
     {
       plugin: HapiOpenAPI,
-      options: isApi ? openAPIOptions : openAdminAPIOptions
+      options: openAPIOptions[service]
     }
   ])
   await server.ext([
     {
       type: 'onPreHandler',
       method: (request, h) => {
-        RequestLogger.logResponse(request)
+        // Perform content-type and accept header validation on API routes- not on admin routes,
```

Comment:
I would similarly suggest that this is added as a plugin to the central-services-shared as a Hapi plugin.